### PR TITLE
VBA Docs Self Service

### PR DESCRIPTION
**Pull request by ATeal**
## Description of change
In order to help users onboard better. We've implemented an endpoint that will allow the user to "see what the server sees". It's gated by a setting that we'll only enable in dev and staging.

resolves https://github.com/department-of-veterans-affairs/vets-contrib/issues/2059

## Testing done
- rspec
- local requests

#### Unique to this PR
- adds ruby zip gem

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by dhall** [comment]
feels like you could push the download logic down into something shared between the controllers

File: modules/vba_documents/app/swagger/vba_documents/v0/controller_swagger.rb [hunk 1]
```diff
@@ -137,6 +137,47 @@ class ControllerSwagger
         end
       end
 
+      swagger_path '/uploads/{id}/download' do
+        operation :get do
+          key :summary, 'Download zip of "what the server sees"'
```
Comment (bastosmichael) [inline modules/vba_documents/app/swagger/vba_documents/v0/controller_swagger.rb]: If we are going to limit this in production then we should include it in the documentation.

File: modules/vba_documents/app/swagger/vba_documents/v1/controller_swagger.rb [hunk 2]
```diff
@@ -165,6 +165,47 @@ class ControllerSwagger
         end
       end
 
+      swagger_path '/uploads/{id}/download' do
+        operation :get do
+          key :summary, 'Download zip of "what the server sees"'
```
Comment (bastosmichael) [inline modules/vba_documents/app/swagger/vba_documents/v1/controller_swagger.rb]: If we are going to limit this in production then we should include it in the documentation.

Comment (ATeal) [reply modules/vba_documents/app/swagger/vba_documents/v1/controller_swagger.rb]: resolved in a7df32a

File: modules/vba_documents/app/swagger/vba_documents/v0/controller_swagger.rb (hunk 1 above)
Comment (ATeal) [reply modules/vba_documents/app/swagger/vba_documents/v0/controller_swagger.rb]: resolved in a7df32a

**Pull request merged**